src/test/ui/const-len-underflow-separate-spans.stderr

```diff
@@ -6,14 +6,23 @@ LL | const LEN: usize = ONE - TWO;
    |
    = note: #[deny(const_err)] on by default
 
-error[E0080]: constant evaluation error
-  --> $DIR/const-len-underflow-separate-spans.rs:17:20
+error: referenced constant
+  --> $DIR/const-len-underflow-separate-spans.rs:21:17
    |
 LL | const LEN: usize = ONE - TWO;
-   |                    ^^^^^^^^^ attempt to subtract with overflow
+   |                    --------- attempt to subtract with overflow
+...
+LL |     let a: [i8; LEN] = unimplemented!();
+   |                 ^^^
+
+error: this expression will panic at runtime
+  --> $DIR/const-len-underflow-separate-spans.rs:21:17
+   |
+LL |     let a: [i8; LEN] = unimplemented!();
+   |                 ^^^ referenced constant has errors
```

Comment:
This seems weird, since the constant is *not* in an `Operand::Const`, but rather in a type.

Reply:
This note means to say that `LEN` has errors, which is a constant reference right where the span points.

Reply:
This is an "error: this expression will panic at runtime" inside non-runtime code though?

Reply:
Oh... Looks like const prop is running on array lengths?! I wonder how that happened.

Reply:
Fixed